display_list/testing/dl_rendering_unittests.cc

```diff
@@ -818,11 +816,34 @@ class TestParameters {
   TestParameters(const SkRenderer& sk_renderer,
                  const DlRenderer& dl_renderer,
                  const DisplayListAttributeFlags& flags)
-      : sk_renderer_(sk_renderer), dl_renderer_(dl_renderer), flags_(flags) {}
+      : TestParameters(sk_renderer, dl_renderer, dl_renderer, flags) {}
+
+  TestParameters(const SkRenderer& sk_renderer,
+                 const DlRenderer& dl_renderer,
+                 const DlRenderer& imp_renderer,
+                 const DisplayListAttributeFlags& flags)
+      : sk_renderer_(sk_renderer),
+        dl_renderer_(dl_renderer),
+        imp_renderer_(imp_renderer),
+        flags_(flags) {}
 
   bool uses_paint() const { return !flags_.ignores_paint(); }
   bool uses_gradient() const { return flags_.applies_shader(); }
 
+  bool impeller_compatible(const DlPaint& paint) const {
+    if (is_draw_text_blob()) {
+      // Non-color text is rendered as paths
+      if (paint.getColorSourcePtr() && !paint.getColorSourcePtr()->asColor()) {
+        return false;
+      }
+      // Non-filled text (stroke or stroke and fill) is rendered as paths
```

Comment:
AFAIK stroke & fill is not exposed at the dart:ui layer, unless there is an indirect path to enabling it.

Reply:
Correct, but with respect to the enum being queried, `!= kFill` is correct. At some point we can eliminate the extra value, but this comparison will still be correct.